### PR TITLE
[Keyboard] Work Louder board fixup

### PR DESCRIPTION
**Pull request by drashna**
## Description

Changes based on feedback and hands on hardware. 

## Types of Changes

- [x] Keyboard (addition or update)

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/work_louder/rgb_functions.c [hunk 1]
```diff
@@ -17,13 +17,160 @@
 #include QMK_KEYBOARD_H
 #include "rgb_functions.h"
 
-#if defined(RGBLIGHT_ENABLE) && defined(RGB_MATRIX_EANBLE)
-#    undef RGB_DI_PIN
-#    define RGBLIGHT_DI_PIN
-#    include "ws2812.c"
+#ifdef RGBLIGHT_ENABLE
```
Comment (fauxpark) [inline keyboards/work_louder/rgb_functions.c:20]: ???

Comment (drashna) [reply keyboards/work_louder/rgb_functions.c:20]: It uses ws2812 on 2 different pins. And one for underglow and one for rgb matrix.

I tried a hack that didn't really work

Comment (fauxpark) [reply keyboards/work_louder/rgb_functions.c:20]: /cry

**Review by fauxpark — APPROVED** [review]

File: keyboards/work_louder/rgb_functions.c (hunk 1 above)
Comment (drashna) [reply keyboards/work_louder/rgb_functions.c:20]: Tell me about it :(

board was already designed.  I'm just happy that it works fine, with no apparent issues

**Pull request merged**